.changeset/context-controller-cavalcade.md

```diff
@@ -0,0 +1,26 @@
+---
+"@patternfly/pfe-core": major
+---
+
+BREAKING CHANGES:
+- Deprecates `context` attribute,
+  use `color` instead or set `--context` css custom property
+- Renames `ContextTheme` type to `ColorTheme`
+- Renames `ColorTheme` type to `ColorPalette`
+- Removes `pfe-contexts` mixin
+
+```html
```

Comment:
label these as before and after

Reply:
Currently `pfe-label` and `pfe-icon` use the `color` attribute explicitly setting a color value which is not part of the context API.  In order to keep consistency across the API, I suggest we use `color-theme` or `color-palette`  instead of `color` or we would have to change the API for those components.

Reply:
@heyMP i tend to agree, `color` is a bit overloaded here. WDYT?

Reply:
Is it possible mark the `color` property as deprecated but not remove it yet?  I'm concerned that we've used it a bunch throughout our CMS.

Reply:
yeah i think we could do that, but we should test for it as well

has the cms updated to pfe2 yet? @zhawkins WDYT?

Reply:
@heyMP if we take your suggestion from below to require context to be explicitly defined on each element then we can implement a deprecated `color` attr only for card and band, does that sound good?